### ISSUE TITLE
Regarding the requirements for the GPU

### ISSUE THREAD

**Issue by Nemophilist98** [uzh-rpg/rampvo#4]
Dear author, I'd like to ask whether the RTX 4090 with 24GB of memory meets the requirements for training? Is the RTX 8000 you are using equipped with 48GB of memory?

**Comment by senecobis** [comment]
Hi @doujiarui.
We indeed use RTX 8000 with 48GB.

The model itself occupies around 20% of the total memory consumption, hence in theory you can fit RAMP in your RTX4090.
The largest part of memory is occupied by gradients and raw event data. You can give it a try with your GPU, but the computation would probably be slower. 

To lower your memory requirement during training, you can change the parameter in **ramp_vo/config_net/your_config** 
` "n_frames": 15, `. This parameter tells how many images every step uses. Since we supervise in correspondence of every image, it means that your backpropagated trajectory is 15 poses long.
If you decide to lower `n_frames` you can probably save a lot of memory but you can expect a slight decrease of performances.

**Comment by Nemophilist98** [comment]
Thank you very much for your reply! Then I'll consider reducing "n_frames" and also think about distributed training (increasing the batch size). I have already successfully carried out distributed training on DPVO before (mainly to reduce the training time), but it seems that the performance has declined, even when using a single GPU. Thanks very much!
